lib/foreman/exception.rb

```diff
@@ -52,7 +52,7 @@ def message
       super unless @wrapped_exception.present?
 
       cls = @wrapped_exception.class.name
-      msg = @wrapped_exception.message.try(:truncate, 90)
+      msg = @wrapped_exception.message
       super + " ([#{cls}]: #{msg})"
```

Comment:
Nitpick - string `Foreman::WrappedException` is very long and it's eating valuable "visible space" on page, I suggest to drop showing class name if it is one of `Exception`, `RuntimeException`, `Foreman::Exception` or `Foreman::WrappedException`. Also the class name could be after the actual message.

A change would be necessary here and in the wrapped exception as well I think.

Reply:
I would prefer not changing format of the message, since it can be parsed by user scripts and this will break it.
Maybe we can omit wrapping messages that inherit from `ForemanException` completely - just pass through the original message - it already contains meaningful information.
What do you think @lzap?

Reply:
Omitting wrapping is killing it's main purpose, please no. Let's keep the message as is then.